### PR TITLE
release 1.1.0

### PR DESCRIPTION
**Pull request by rohitthakur2590**
Signed-off-by: Rohit Thakur <rohitthakur2590@outlook.com>

##### SUMMARY

Regenerate docs
Link changelog to README
Prepare for 1.1.0 release

##### ISSUE TYPE

- Docs Pull Request

##### ADDITIONAL INFORMATION

```paste below

```

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -25,6 +25,10 @@ Name | Description
 --- | ---
 [cisco.iosxr.iosxr](https://github.com/ansible-collections/cisco.iosxr/blob/main/docs/cisco.iosxr.iosxr_cliconf.rst)|Use iosxr cliconf to run command on Cisco IOS XR platform
 
+### Filter plugins
```
Comment (NilashishC) [inline README.md:28]: filter/ and inventory/ dirs got generated by cli_rm_builder, but are actually empty. We can remove these sections.

File: changelogs/CHANGELOG.rst [hunk 2]
```diff
@@ -5,6 +5,24 @@ Cisco Iosxr Collection Release Notes
 .. contents:: Topics
 
 
+v1.1.0
+======
+
+Major Changes
```
Comment (NilashishC) [inline changelogs/CHANGELOG.rst:11]: It'll be a minor change.

Comment (NilashishC) [reply changelogs/CHANGELOG.rst:11]: As discussed, major changes involve backward compatible ones and need a major version bump.

File: changelogs/changelog.yaml [hunk 3]
```diff
@@ -125,3 +125,17 @@ releases:
     - 64-incorrect-interface-decription-parsing-fix.yaml
     - iosxr_netconf_config_commit_fix.yaml
     release_date: '2020-08-28'
+  1.1.0:
+    changes:
+      major_changes:
```
Comment (NilashishC) [inline changelogs/changelog.yaml:130]: Same as above.

File: README.md (hunk 1 above)
Comment (rohitthakur2590) [reply README.md:28]: yes that make sense

**Review by NilashishC — APPROVED** [review]

**Pull request merged**